integration/kubernetes/confidential/sev.bats

```diff
@@ -302,240 +51,235 @@ setup_file() {
+  esudo rm -rf "${TEST_DIR}"
 }
 
 setup() {
-  # Remove the service/deployment/pod if it exists
-  echo "Deleting previous test pods..."
-  delete_pods
-
-  # Delete any previous data in the DB
-  mysql -u${KBS_DB_USER} -p${KBS_DB_PW} -h ${KBS_DB_HOST} -D ${KBS_DB} <<EOF
-    DELETE FROM secrets WHERE id = 10;
-    DELETE FROM keysets WHERE id = 10;
-    DELETE FROM policy WHERE id = 10;
-EOF
+  # Remove any previous k8s test services
+  echo "Deleting previous test services..."
+  k8s_delete_all
+
+  # Delete any previous data in the simple-kbs database
+  simple_kbs_delete_data
 }
 
-@test "$test_tag Test SEV unencrypted container launch success" {
+
+@test "${TEST_TAG} Test SEV unencrypted container launch success" {
   # Turn off pre-attestation. It is not necessary for an unencrypted image.
-  esudo sed -i 's/guest_pre_attestation = true/guest_pre_attestation = false/g' ${SEV_CONFIG}
+  esudo sed -i 's/guest_pre_attestation = true/guest_pre_attestation = false/g' ${SEV_CONFIG_FILE}
   
   # Start the service/deployment/pod
-  esudo kubectl apply -f "${TEST_DIR}/unencrypted-image-tests.yaml"
+  esudo kubectl apply -f "${TEST_DIR}/sev-unencrypted.yaml"
   
   # Retrieve pod name, wait for it to come up, retrieve pod ip
-  pod_name=$(esudo kubectl get pod -o wide | grep unencrypted-image-tests | awk '{print $1;}')
-  kubernetes_wait_for_pod_ready_state "$pod_name" 20
-  pod_ip=$(esudo kubectl get pod -o wide | grep unencrypted-image-tests | awk '{print $6;}')
+  local pod_name=$(esudo kubectl get pod -o wide | grep sev-unencrypted | awk '{print $1;}')
+  k8s_wait_for_pod_ready_state "$pod_name" 20
+  local pod_ip=$(esudo kubectl get pod -o wide | grep sev-unencrypted | awk '{print $6;}')
 
-  print_service_info
+  k8s_print_info "sev-unencrypted"
 
   # Look for SEV enabled in container dmesg output
-  sev_enabled=$(ssh -i ${SSH_KEY_FILE} \
-    -o "StrictHostKeyChecking no" \
-    -o "PasswordAuthentication=no" \
-    -t root@${pod_ip} \
-    'dmesg | grep SEV' || true)
-
-  if [ -z "$sev_enabled" ]; then
-    >&2 echo -e "${RED}KATA CC TEST - FAIL: SEV is NOT Enabled${NC}"
+  local sev_enabled=$(ssh_dmesg_grep \
+    "${SSH_KEY_FILE}" \
+    "${pod_ip}" \
+    "${SEV_DMESG_GREP_TEXT}")
+
+  if [ -z "${sev_enabled}" ]; then
+    >&2 echo -e "KATA SEV TEST - FAIL: SEV is NOT Enabled"
     return 1
   else
-    echo "DMESG REPORT: $sev_enabled"
-    echo -e "${GREEN}KATA CC TEST - PASS: SEV is Enabled${NC}"
+    echo "DMESG REPORT: ${sev_enabled}"
+    echo -e "KATA SEV TEST - PASS: SEV is Enabled"
   fi
-
 }
 
-@test "$test_tag Test SEV encrypted container launch failure with INVALID measurement" {
+@test "${TEST_TAG} Test SEV encrypted container launch failure with INVALID measurement" {
   # Make sure pre-attestation is enabled. 
-  esudo sed -i 's/guest_pre_attestation = false/guest_pre_attestation = true/g' ${SEV_CONFIG}
+  esudo sed -i 's/guest_pre_attestation = false/guest_pre_attestation = true/g' ${SEV_CONFIG_FILE}
```

Comment:
Then you don't need to make this change because you didn't touch the config file. :)

Reply:
Used pod annotations.